test/test_plugin_twitter.py

```diff
@@ -945,12 +1032,11 @@ def test_plugin_twitter_tweet_attachments(mock_get, mock_post):
     ]
 
     # We'll fail to send this time
+    obj = Apprise.instantiate(twitter_url + "?mode=tweet")
```

Comment:
This shouldn't be here. The test is to call `notify` back to back and verify that subsequent calls leverage some internal cached values (preventing extra get/post requests)

Reply:
There is a dedicated test case for verifying the caching now.

https://github.com/caronc/apprise/blob/1e960c0b9a1fe5a5570292994e33ec28989f7a24/test/test_plugin_twitter.py#L526-L578

Reply:
12091a0 improves this test case a bit further, by also explicitly validating the content of the cache slots.

https://github.com/caronc/apprise/blob/12091a0995241ce19dc147c7493ddb8284c8adda/test/test_plugin_twitter.py#L566-L570

Reply:
After rebasing, it looks like this very safeguard check (0fe4e34f1) just catched a potential regression introduced by #750?

The test which validates the cache content, fails exactly there.

-- https://github.com/caronc/apprise/actions/runs/3466494672/jobs/5790409732#step:10:502

Reply:
I've decided to pull this into a different patch, see https://github.com/caronc/apprise/pull/728#issuecomment-1329417002.